joomla/joomla-cms
# Proxy old phputf8 library path to new

#### Summary of Changes

This PR proxies the phputf8 library files located at `libraries/phputf8` to their counterparts at `libraries/vendor/joomla/string/src/phputf8` and removes all of the declared functions from these files.  Short of someone including this library themselves in yet another location, this should solve any reports dealing with its functions being redeclared.
#### Testing Instructions

To validate the proxy, add the following line to your template's index.php file:

``` php
require_once JPATH_PLATFORM . '/phputf8/mbstring/core.php';
```

This will import the file from the old location and should cause a deprecation log message telling you to use the new location.

Otherwise as there are no core uses of the old file path, things should just work.
#### Why `class_exists()` Checks on Logging?

This is a third party API and is 100% usable outside the context of Joomla.  The `class_exists()` checks keep it this way.
